node/src/types/deploy.rs

```diff
@@ -69,11 +63,11 @@ static DEPLOY: Lazy<Deploy> = Lazy::new(|| {
         ttl: TimeDiff::from(3_600_000),
         gas_price: 1,
         body_hash,
-        dependencies: vec![DeployHash::new(Digest::from([1u8; Digest::LENGTH]))],
+        dependencies: vec![DeployHash::new(Digest::hash([1u8; Digest::LENGTH]))],
```

Comment:
`Digest::from` didn't hash the input, did it?

Reply:
No, it didn't. I think the hashed version is more appropriate in the deploy example above.